cloudhub-2/modules/ROOT/pages/ps-manage.adoc

```diff
@@ -137,6 +137,29 @@ include::partial$select-private-space.adoc[tag=clickPrivateSpaceName]
 // image::vpn-view-details.png[View Details option in the VPN menu]
 . Click *X* to close the *VPN Details* window.
 
+[[enable-vpn-logs]]
+== Enable VPN logs
+
+You can enable VPN logs to be directly accessible by MuleSoft support engineers. This shortens the process for obtaining the logs from AWS.
+
+. For VPN connections created after March 8, 2023, VPN logs are enabled by default upon creation.
+. For existing VPN connections:
+.. You can enable VPN logs via PATCH call to the following existing public endpoint:
++
+`https://{{env}}.anypoint.mulesoft.com/runtimefabric/api/organizations/{{org-id}}/privatespaces/{{ps-id}}/connections/{{connection-id}}/vpns/{{vpn-id}}`.
```

Comment:
actually, do we always leave the {{env}} when sharing an API endpoint? For customers, this {{env}} can only be either `eu1` (for prod-eu control plane) or nothing (for prod) @luanamulesoft

Reply:
I initially put the `env` there for my testing purpose to test in different envs